lib/TFD1770_refactor.rb

```diff
@@ -0,0 +1,28 @@
+module Instance_method
```

Comment:
- It will be better if you rename this file to a reasonable name, instead of `TFD1770_refactor.rb`.
- Also, you may need to put these two modules into two lib files since they are quite different.
- Also rename the module names to reasonable names, instead of `Instance_method` and `Class_method`.

Reply:
I will fix them ASAP, Thank you!

Reply:
I have finished all the modifications except deleting "require", because if I delete it, the test will not be passed. Can you explain this why I cannot delete it?